### DISCUSSION TITLE
Feedback Feature: Get an user feedback text in a office form

### DISCUSSION THREAD

**Discussion by Marlony-Gabriel** [squidfunk/mkdocs-material#5337] [Q&A] [answered]
Thats my mkdocs.yml for feedback feature: 

```
extra:
  analytics:
    provider: google
    property: G-ZGH4E6984C
    feedback:
      title: Was this page helpful?
      ratings:
        - icon: material/emoticon-happy-outline
          name: This page was helpful
          data: 1
          note: >-
            Thanks for your feedback!
        - icon: material/emoticon-sad-outline
          name: This page could be improved
          data: 0
          note: >- 
            Thanks for your feedback! Help us improve this page by
            using our <a href="..." target="_blank" rel="noopener">feedback form</a>.
  consent:
    title: Cookie consent
    description: >- 
      We use cookies to recognize your repeated visits and preferences, as well
      as to measure the effectiveness of our documentation and whether users
      find what they're searching for. With your consent, you're helping us to
      make our documentation better.
```
When an user give a negative feedback, I want to show an office form with a text field, and ask the users to tell me what they don't like. This part is ok. I already had the script to do it. I also would like to save in the form  the url where the users are filling this text field. Is it possible, using a office form ?

**Comment by squidfunk** [comment, marked as answer]
You can put any link inside the `href`. We're using Google Forms to collect feedback, but you can use any other service, [as explained in our documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-analytics/#+analytics.feedback.ratings.note). Those are the settings we are using:

``` yaml
feedback:
      title: Was this page helpful?
      ratings:
        - icon: material/emoticon-happy-outline
          name: This page was helpful
          data: 1
          note: Thanks for your feedback!
        - icon: material/emoticon-sad-outline
          name: This page could be improved
          data: 0
          note: >-
            Thanks for your feedback! Help us improve this page by
            <a href="https://docs.google.com/forms/d/e/1FAIpQLSe8L5YDggZiVjgvBKfv-XCdbQAEJrseRh8MBh-jS92TU2aqzw/viewform?usp=pp_url&entry.853143515={url}" target=_blank>telling us what you're missing</a>.
```

**Reply by Marlony-Gabriel** [reply]
Thank you for the answer @squidfunk . Thank you so much.
I'm trying to do something like this, but with a microsoft form. https://forms.office.com/r/Qv5Vb1z055
But i haven't found a solution yet.
Just in case I have to change to a google form, this information: "usp=pp_url&entry.853143515", where/how you get it ?
Again, thank you very much to take a time to answer me. I really apreciate all the hard work you made here. Material is an extraordinary theme to mkdocs.
